state-sync/src/request_manager.rs

```diff
@@ -326,6 +324,22 @@ impl RequestManager {
         }
     }
 
+    fn get_network_sender(&mut self, peer: &PeerNetworkId) -> StateSyncSender {
+        self.network_senders
+            .get_mut(&peer.network_id())
+            .unwrap_or_else(|| panic!("Missing network sender for peer: {:?}", peer))
+            .clone()
+    }
+
+    pub fn send_chunk_response(
+        &mut self,
+        peer: &PeerNetworkId,
+        message: StateSyncMessage,
+    ) -> Result<(), NetworkError> {
+        let mut network_sender = self.get_network_sender(peer);
+        network_sender.send_to(peer.peer_id(), message)
```

Comment:
Let's one line it!

Reply:
Done, but the formatter turns it into 2 lines 💩